### ISSUE TITLE
Cannot invoke "com.fastasyncworldedit.core.queue.IQueueExtent.getCachedGet(int, int)" because "this.extent" is null

### ISSUE THREAD

**Issue by TheLimeGlass** [IntellectualSites/FastAsyncWorldEdit#2553]
### Server Implementation

Paper

### Server Version

1.20

### Describe the bug

Simple error fix because it's a null pointer exception.

### To Reproduce

running command `//r ~h cyan_terracotta` caused error.

### Expected behaviour

To not throw a null pointer exception

### Screenshots / Videos

```
[FAWE Fork Join Pool Primary - 6/ERROR]: [com.fastasyncworldedit.core.queue.implementation.ParallelQueueExtent] Catching
java.lang.NullPointerException: Cannot invoke "com.fastasyncworldedit.core.queue.IQueueExtent.getCachedGet(int, int)" because "this.extent" is null
	at com.fastasyncworldedit.core.queue.implementation.chunk.ChunkHolder.newWrappedGet(ChunkHolder.java:992) ~[FastAsyncWorldEdit-Bukkit-2.8.5-SNAPSHOT.jar:?]
	at com.fastasyncworldedit.core.queue.implementation.chunk.ChunkHolder.getOrCreateGet(ChunkHolder.java:961) ~[FastAsyncWorldEdit-Bukkit-2.8.5-SNAPSHOT.jar:?]
	at com.fastasyncworldedit.core.queue.implementation.chunk.ChunkHolder$4.getBlock(ChunkHolder.java:762) ~[FastAsyncWorldEdit-Bukkit-2.8.5-SNAPSHOT.jar:?]
	at com.fastasyncworldedit.core.queue.implementation.chunk.ChunkHolder.getBlock(ChunkHolder.java:1087) ~[FastAsyncWorldEdit-Bukkit-2.8.5-SNAPSHOT.jar:?]
	at com.fastasyncworldedit.core.queue.IChunkExtent.getBlock(IChunkExtent.java:62) ~[FastAsyncWorldEdit-Bukkit-2.8.5-SNAPSHOT.jar:?]
	at com.sk89q.worldedit.extent.InputExtent.getBlock(InputExtent.java:53) ~[FastAsyncWorldEdit-Bukkit-2.8.5-SNAPSHOT.jar:?]
	at com.fastasyncworldedit.core.extent.PassthroughExtent.getBlock(PassthroughExtent.java:194) ~[FastAsyncWorldEdit-Bukkit-2.8.5-SNAPSHOT.jar:?]
	at com.fastasyncworldedit.core.extent.PassthroughExtent.getBlock(PassthroughExtent.java:194) ~[FastAsyncWorldEdit-Bukkit-2.8.5-SNAPSHOT.jar:?]
	at com.fastasyncworldedit.core.function.mask.SingleBlockStateMask.test(SingleBlockStateMask.java:33) ~[FastAsyncWorldEdit-Bukkit-2.8.5-SNAPSHOT.jar:?]
	at com.fastasyncworldedit.core.function.mask.CachedMask.test(CachedMask.java:71) ~[FastAsyncWorldEdit-Bukkit-2.8.5-SNAPSHOT.jar:?]
	at com.fastasyncworldedit.core.function.mask.AdjacentAnyMask.direction(AdjacentAnyMask.java:58) ~[FastAsyncWorldEdit-Bukkit-2.8.5-SNAPSHOT.jar:?]
	at com.fastasyncworldedit.core.function.mask.AdjacentAnyMask.test(AdjacentAnyMask.java:43) ~[FastAsyncWorldEdit-Bukkit-2.8.5-SNAPSHOT.jar:?]
	at com.fastasyncworldedit.core.extent.filter.MaskFilter.applyBlock(MaskFilter.java:38) ~[FastAsyncWorldEdit-Bukkit-2.8.5-SNAPSHOT.jar:?]
	at com.fastasyncworldedit.core.extent.filter.block.CharFilterBlock.filter(CharFilterBlock.java:176) ~[FastAsyncWorldEdit-Bukkit-2.8.5-SNAPSHOT.jar:?]
	at com.sk89q.worldedit.regions.Region.filter(Region.java:327) ~[FastAsyncWorldEdit-Bukkit-2.8.5-SNAPSHOT.jar:?]
	at com.sk89q.worldedit.regions.Region.filter(Region.java:310) ~[FastAsyncWorldEdit-Bukkit-2.8.5-SNAPSHOT.jar:?]
	at com.sk89q.worldedit.regions.CuboidRegion.filter(CuboidRegion.java:762) ~[FastAsyncWorldEdit-Bukkit-2.8.5-SNAPSHOT.jar:?]
	at com.fastasyncworldedit.core.extent.filter.block.ChunkFilterBlock.filter(ChunkFilterBlock.java:88) ~[FastAsyncWorldEdit-Bukkit-2.8.5-SNAPSHOT.jar:?]
	at com.fastasyncworldedit.core.queue.implementation.chunk.ChunkHolder.filterBlocks(ChunkHolder.java:910) ~[FastAsyncWorldEdit-Bukkit-2.8.5-SNAPSHOT.jar:?]
	at com.fastasyncworldedit.core.queue.IQueueExtent.apply(IQueueExtent.java:144) ~[FastAsyncWorldEdit-Bukkit-2.8.5-SNAPSHOT.jar:?]
	at com.fastasyncworldedit.core.queue.implementation.ParallelQueueExtent.lambda$apply$0(ParallelQueueExtent.java:133) ~[FastAsyncWorldEdit-Bukkit-2.8.5-SNAPSHOT.jar:?]
	at java.util.concurrent.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1375) ~[?:?]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) ~[?:?]
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[?:?]
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[?:?]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[?:?]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[?:?]
```

### Fawe Debugpaste

https://www.athion.net/ISPaster/paste/view/a2d6bddb4a0d46e4ba144a36ff1e0beb

### Fawe Version

2.8.5-snapshot

### Checklist

- [X] I have included a Fawe debugpaste.
- [X] I am using the newest build from https://ci.athion.net/job/FastAsyncWorldEdit/ and the issue still persists.

**Comment by OneLiteFeather** [comment]
@TheLimeGlass the debugpaste link is not working, can you generate a new one?

**Comment by dordsor21** [comment]
Fixed by #2594
